templates/views/events.html

```diff
@@ -150,9 +153,44 @@ <h5>{{event.location}}</h5>
                 </div>
             </div>
             {% else %}
-            <div class="col-sm-10 col-sm-offset-2 col-md-11 col-md-offset-1">
-                <h4 class="text-muted">There are currently no available events postings</h4>
-            </div>
+            <div class="row">
+            <p class="__shecodeval1">PAST EVENTS</p><br>
```

Comment:
@maryam-olabisi Past events here should be black (Past) and SCA colour (Events) just as we have 'Our Team' on the teams page